db/migrate/20241111133654_create_discourse_activity_pub_clients.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class CreateDiscourseActivityPubClients < ActiveRecord::Migration[7.1]
+  def change
+    create_table :discourse_activity_pub_clients do |t|
+      t.integer :auth_type, null: false
+      t.string :domain, null: false
```

Comment:
Let's add a length limit to the integer and string columns here. 1000 chars should be enough for both?

Reply:
I've added limits to the string columns in the migrations: https://github.com/angusmcleod/discourse-activity-pub/commit/dcf1e5eef381140ce5f9354441d4d118a5af4153. Is a limit on the `auth_type` int column necessary?

Reply:
No need for a limit on int column... don't know what I was thinking!